grizzly/users/messagequeue.py

```diff
@@ -197,14 +213,23 @@ def __init__(self, environment: Environment, *args: Tuple[Any, ...], **kwargs: D
 
         auth_context = self._context.get('auth', {})
         username = auth_context.get('username', None)
+        message_context = self._context.get('message', {})
+        header_type = message_context.get('header_type', None)
+        if header_type:
+            header_type = header_type.lower()
+            if header_type not in ['rfh2', 'none']:
```

Comment:
where is `header_type` set to `"none"` ? 
line 218, if value is the default (as specified on line 158) would be `None` (`NoneType`), would not get down to this check? my understanding is that `None` would be the same as `"none"`.

Reply:
The message.header_type context variable can be set to either "RFH2" or "none" (or be left out), where "none" would mean the same as not specifying header_type at all.

Reply:
this.